services/minespace-web/src/components/dashboard/mine/reports/ReportsTable.js

```diff
@@ -46,7 +46,11 @@ const columns = [
     dataIndex: "due_date",
     key: "due_date",
     defaultSortOrder: "ascend",
-    sorter: (a, b) => (moment(a.due_date) > moment(b.due_date) ? -1 : 1),
+    sorter: (a, b) => {
+      const a_date = a.due_date == null ? moment().add(200, "y") : moment(a.due_date);
+      const b_date = b.due_date == null ? moment().add(200, "y") : moment(b.due_date);
+      return a_date - b_date;
```

Comment:
Should we update all date sorters to handle null?

Reply:
I turned this into a util function that can be applied wherever we need.